src/app/utils/calls.js

```diff
@@ -56,9 +57,9 @@ export const postDataForValidation = data => {
 
   return fetch(url, myInit)
     .then(res => {
-      if (! res.ok) {
+      if ((! res.ok) && (res.status != 422)) {
```

Comment:
And here.

The use of 422 to indicate a validation error is super awkward and IMO wrong. I added https://github.com/italia/publiccode-validator/issues/17 to document what the validator is supposed to return in the first place and I think we can also look into changing the current response as a next step.

If we just want a quick fix, this can do it for now.

Reply:
This was discusses long time ago, and for sure we can review it. The approach is described in openapi specs: https://github.com/italia/publiccode-validator/blob/master/api_spec_oa3.yml